### ISSUE TITLE
Cybs.ini should be picked up from current or configurable directory

### ISSUE THREAD

**Issue by brianmc** [CyberSource/cybersource-sdk-php#19]
Having cybs.ini inside the sdk itself is great for tests and built-in samples but you really want to have the configuration within your application.  In addition if you use something like Azure/heroku auto-deploys you really can't change anything in the SDK/vendor dirs.

What I was able to do to work around was just set the include path to the current dir, where my config was and maybe that's enough if it is documented.  

set_include_path(getcwd());

**Comment by thedancingpanda** [comment]
I would say the better workaround is to write a class that extends CybsClient and then write your own constructor. This also allows you to have more than one CyberSource account if need be.

**Comment by jrgonzalezrios** [comment]
@thwiv That worked for me, thanks

**Comment by gmoros-dev** [comment]
> 
> 
> I would say the better workaround is to write a class that extends CybsClient and then write your own constructor. This also allows you to have more than one CyberSource account if need be.

I agree with @thwiv, the package sould support manage more than one business account
